luna/featurevis/images.py

```diff
@@ -36,30 +36,26 @@ def deprocess_image(img):
     """
     Takes the values of an image array and normalizes them to be in the
     standard 0-255 RGB range
-
     :param img: The generated image array
-
     :return: A rescaled version of the image
     """
     print('Deprocessing image')
-    # Normalize array between 0 and 1
-    img = (img - np.min(img)) / (np.max(img) - np.min(img))
+    img = ((img - img.mean()) / img.std()) + 1e-5
+    img *= 0.15
+    img += 0.5
+    img = np.clip(img, 0, 1)
 
-    # Convert to RGB array
     img *= 255
-    img = img.astype("uint8")
+    img = np.clip(img, 0, 255).astype("uint8")
```

Comment:
Can you add comments on what happens here? The one in the function doctoring seems not to cover this anymore.